coreneuron/io/mk_mech.cpp

```diff
@@ -145,7 +145,7 @@ static void mk_mech(std::istream& s) {
         fflush(stderr);
     }
     /* will have to put this back if any mod file refers to diam */
-    //	register_mech(morph_mech, morph_alloc, (Pfri)0, (Pfri)0, (Pfri)0, (Pfri)0, -1, 0);
+    //	register_mech(morph_mech, morph_alloc, (int(*)())0, (Pfri)0, (Pfri)0, (Pfri)0, -1, 0);
```

Comment:
where is `Pfri` declared?

Reply:
I remove those changes because it needs fix inside mod2c and nmodl before